project/src/components/add-form-review/add-form-review.tsx

```diff
@@ -1,18 +1,33 @@
 import {ReviewsType} from '../../types/property';
 import { AddReviewsList } from '../reviews-list/reviews-list';
-import {useState} from 'react';
+import {useState, FormEvent} from 'react';
+import {useAppSelector, useAppDispatch} from '../../hooks';
+import { useParams } from 'react-router-dom';
+import { fetchPostReviewAction } from '../../store/api-actions';
+import { MAX_REVIEW_LENGTH, MIN_REVIEW_LENGTH } from '../../const';
 
-type AddFormReviewProps = {
-  reviews: ReviewsType[];
-};
+function AddFormReview (): JSX.Element{
+  const reviews = useAppSelector((store) => store.reviews);
```

Comment:
```suggestion
    const { reviews} = useAppSelector((state) => state);
```
а можно вот так писать?

Reply:
{ reviews } это деструктуризация, когда мы знаем что reviews уже имеет тип ReviewsType. Я так понял мы знаем что в хранилище уже лежит reviews и мы можем сразу менять состояния, когда это нужно через useAppSelector